components/purchase_modal/purchase_modal.tsx

```diff
@@ -127,6 +146,76 @@ function getSelectedProduct(products: Record<string, Product> | undefined, produ
     }
     return findProductInDictionary(products, null, nextSku);
 }
+
+type PlanLabelProps = {
+    text: string;
+    bgColor: string;
+    color: string;
+    firstSvg: JSX.Element;
+    secondSvg?: JSX.Element;
+}
+
+function PlanLabel(props: PlanLabelProps) {
+    return (
+        <div
+            className='planLabel'
+            style={{
+                backgroundColor: props.bgColor,
+                color: props.color,
+            }}
+        >
+            {props.firstSvg}
+            {props.text}
+            {props.secondSvg}
+        </div>
+    );
+}
+
+function Card(props: CardProps) {
```

Comment:
Ditto on extracting the `Card` in this component and in `components/pricing_modal/content.tsx`.